app.js

```diff
@@ -113,6 +113,8 @@ global.Ladders = require(Config.remoteladder ? './ladders-remote.js' : './ladder
 
 global.Users = require('./users.js');
 
+global.Punishments = require('./punishments.js');
```

Comment:
If the below checks are moved into punishments.js, then it can just be required in users. A bit convoluted with the current code, though.